Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM alpine
 
-ENV DATA_DIRECTORY /data
-ENV PASSWORD_FILE /data/.htpasswd
+ENV DATA_DIRECTORY="/data"
+ENV PASSWORD_FILE="/data/.htpasswd"
+ENV DISABLE_AUTHENTICATION=""
```

Comment:
Is it necessary to predefine all environement variables? Or is the idea here to make the available ones more visible?

Reply:
Also note that it's possible to reduce the number of layers created by creating the variables in a single `ENV` directive (just remove all but the first `ENV` then add a backslash to each line excluding the last line).

Reply:
Does the number of metadata layers of a docker image matter all? AFAIK the number of layers is only relevant for the filesystem overlays?

Reply:
You're correct. Apparently, it's been that way since Docker 1.10 in 2016 and I never noticed.